tools/nvidia-gpu-lock.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I'm getting some syntax errors from this script:

```
GPU clocks set to "(gpuClkMin 3105, gpuClkMax 3105)" for GPU 00000000:01:00.0
All done.
Memory clocks set to "(memClkMin 9001, memClkMax 9001)" for GPU 00000000:01:00.0
All done.
nvidia-gpu-lock.sh: 12: function: not found
All done.
All done.
nvidia-gpu-lock.sh: 15: Syntax error: "}" unexpected
```

Reply:
Uh, hm. What platform? I can't reproduce this on my end :(

Reply:
Debian Linux

Reply:
Solved the problem: Debian aliases /bin/sh to `dash` not `bash` (which is another posix compliant shell). `function reset {` is a bash-only thing. If I use the /bin/bash binary directly the current script works.

You can increase compatibility by using this syntax (works for both shells):
```
reset() {
    nvidia-smi --reset-gpu-clocks
    nvidia-smi --reset-memory-clocks
}
```